### ISSUE TITLE
UUID conversion error

### ISSUE THREAD

**Issue by max4t** [Raizlabs/DBFlow#1313]
DBFlow Version: 4.0.3
Issue Kind (Bug, Question, Feature): Bug

This bug appears in the generated tables when a foreign key refers to a table whose primary key is a UUID (and maybe other types but i haven't tested)

Here is my table:
```kotlin
data class Transfer(
    @PrimaryKey var transfer_id: UUID = UUID.randomUUID(),
    @ForeignKey(stubbedRelationship = true) var origin: Account? = null,
    @ForeignKey(stubbedRelationship = true) var destination: Account? = null
)
```
The generated table(`Transfer_Table.java`) has an error : 
```kotlin
@Override
public final void loadFromCursor(FlowCursor cursor, Transfer model) {
  ...
  if (index_origin_aid_Account_Table != -1 && !cursor.isNull(index_origin_aid_Account_Table)) {
    model.setOrigin(...);
    model.getOrigin().setAid(cursor.getString(index_origin_aid_Account_Table));
  } else {
    model.setOrigin(null);
  }
  ...
```
In the above sample, this : 
```kotlin
model.getOrigin().setAid(cursor.getString(index_origin_aid_Account_Table));
```
should be this : 
```kotlin
model.getOrigin().setAid(global_typeConverterUUIDConverter.getModelValue(cursor.getString(index_origin_aid_Account_Table)));
```
This was just for the `origin` but the same goes for the `destination`

**Comment by max4t** [comment]
If i'm correct, this error is [here](https://github.com/Raizlabs/DBFlow/blob/develop/dbflow-processor/src/main/java/com/raizlabs/android/dbflow/processor/definition/column/ForeignKeyAccessCombiner.kt#L140) : `.set(cursorAccess, ...` should be `.set(fieldAccessBlock, ...`, right?

**Comment by agrosner** [comment]
fixed in `develop` will be in 4.0.4. if you wish to get it earlier use `522ee9b15a` as your version.

**Comment by max4t** [comment]
Thanks!
